index.js

```diff
@@ -1 +1,22 @@
-// phew… not a lot going on here. Please add some code!
+const bookMarkToggle = document.querySelector('[data-js="bookMark"]');
+
+bookMarkToggle.addEventListener("click", () => {
+bookMarkToggle.classList.toggle("bookmark--active");
+});
+
+const card =document.querySelector('[class="card-list__item"]');
+
+const answerButton =document.querySelector('[class="card__button-answer"]');
+
+const shownAnswer =document.querySelector('[data-js="answerOne"]');
```

Comment:
```suggestion
const answer =document.querySelector('[data-js="answer"]');
```
Just answer would be maybe a bit more suitable, since you toggle it and not only show the answer, or?